build.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/sh
+gradle -Pbuildnumber=bag-tooltip build
```

Comment:
Sorry that's my script to make the jar different from default build e.g. `ActuallyAdditions-1.10.2-r90-bag-tooltip.jar` 

I will remove that file